blog/data/week 8 set up project&Module 7.md

```diff
@@ -0,0 +1,46 @@
+# IMDb project and Module 7 exercises
+In this week we set up the IMDb project and realize the exercises of Module 7.
+## IMDb project
+We did all together a template for organize ourselves and estblish the jobs to do.
+### Module 7 exercises.
+Along this week we did the exercises related with module 7, we learnt how to use properly graphs that allow us to get the insights needed for a good analysis.
+
+
+*09/05/2022*
+| **Path Owner** | **Content Deliverer** |
+| --- | --- | 
+| [Eva Gonzalez](https://github.com/evag-empathy) | [Aida Riesgo](https://github.com/Aidaempathy) | \
+
+\
+&nbsp;
+**Key learning points**
+1. Manage Miro template
+2. how to work in team for a project
+3. How set up a script with the functions needed
+
+At the beginning of the week we started IMDb project from the data point of view.
+
+All the academy´s teams leaded by Jose (https://github.com/jllorian)
+worked together in a Miro template and set up the jobs to do, and who should do them.
+
+We worked also in a Python´s script tha could manage the data and get the insights for the project, within the data team we created a GitHub repository with that purpose.
```

Comment:
- "That could manage", little typo there
- I think "for that purpose" is better that "with that purpose" (not sure about this one)